src/client/components/household-information/FinancialDisclosureSection.jsx

```diff
@@ -45,36 +35,37 @@ class FinancialDisclosureSection extends React.Component {
       content = (<fieldset>
         <legend>Financial Disclosure</legend>
         <div className="input-section">
-          {notRequiredMessage}
-
+          <p>In the next 4 sections you will be asked to provide your income information from the most recent tax year. Your most recent
```

Comment:
Can we add a sentence between these 2 saying "This information is used to determine your eligibility for health care".